server/apiserver/argoserver.go

```diff
@@ -302,7 +303,15 @@ func (as *argoServer) newHTTPServer(ctx context.Context, port int, artifactServe
 	mustRegisterGWHandler(workflowarchivepkg.RegisterArchivedWorkflowServiceHandlerFromEndpoint, ctx, gwmux, endpoint, dialOpts)
 	mustRegisterGWHandler(clusterwftemplatepkg.RegisterClusterWorkflowTemplateServiceHandlerFromEndpoint, ctx, gwmux, endpoint, dialOpts)
 
-	mux.HandleFunc("/api/", func(w http.ResponseWriter, r *http.Request) { webhookInterceptor(w, r, gwmux) })
+	apiHandler := http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
+		webhookInterceptor(rw, r, gwmux)
+	})
+
+	if os.Getenv("ENABLE_SSE_KEEPALIVE") != "" {
```

Comment:
I figured it might be good to ship this initially behind a feature flag before releasing it generally, but would be interested to hear opinions

Reply:
I think on by default rather than off